src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -743,6 +743,9 @@ export default {
 		}
 	},
 	computed: {
+		loanDisplayComponent() {
+			return this.$refs?.mlLoanDisplay?.$refs || this.$refs?.mlLoanDisplay?.[0]?.$refs;
+		},
```

Comment:
Note that computed properties looking at refs will only evaluate on the initial page setup. values nested in refs are not reactive. Should be fine for initially setting the grid view vs carousel view. Just need to make sure toggling works fine.